docs/document/content/user-manual/shardingsphere-jdbc/special-api/transaction/seata.cn.md

```diff
@@ -80,16 +80,16 @@ ALTER TABLE `undo_log` ADD INDEX `ix_log_created` (`log_created`);
 
 `seata.conf` 存在四个属性，
 
-1. `sharding.transaction.seata.at.enable`，当此值为`true`时，开启 ShardingSphere 的 Seata AT 集成，存在默认值为 `true`
-2. `sharding.transaction.seata.tx.timeout`，全局事务超时（秒），存在默认值为 `60`
+1. `shardingSphere.transaction.seata.at.enable`，当此值为`true`时，开启 ShardingSphere 的 Seata AT 集成，存在默认值为 `true`
```

Comment:
Please modify shardingSphere to shardingsphere

Reply:
Thank you very much for your review. I have made the corresponding adjustments.